Felienne/hedy
# Adds research section to learn-more page, dynamically load papers

**Description**
In this PR we add a research section to the learn-more page, containing related research performed on Hedy. The list is dynamically created based on the content in the `/research` folder. When adding a new .pdf file the name is automatically parsed and the corresponding routing is added. This should make it easy to add research for all contributors. We do have a specific naming convention to keep the list structured. One should keep the following approach:

- Name the file the title of your thesis
- Use underscores instead of whitespaces to ensure its a valid filename
- The name is dynamically parsed to look nicely

**Example**
`An_approach_to_describing_the_semantics_of_Hedy.pdf`
Will be shown as:
`An approach to describing the semantics of Hedy`

**Fixes**
This PR fixes #3037.

**How to test**
Navigate to the `/learn-more` page and notice the new research section at the bottom. Verify that the current link (it is only one paper at the moment) works and the .pdf file is shown in a new tab.
